doc/developers/maintainer.rst

```diff
@@ -397,8 +397,8 @@ experimental features / estimators that are subject to change without
 deprecation cycle.
 
 To create an experimental module, you can just copy and modify the content of
-`enable_hist_gradient_boosting.py
-<https://github.com/scikit-learn/scikit-learn/blob/c9c89cfc85dd8dfefd7921c16c87327d03140a06/sklearn/experimental/enable_hist_gradient_boosting.py>`__,
+`enable_halving_search_cv.py
+<https://github.com/scikit-learn/scikit-learn/blob/362cb92bb2f5b878229ea4f59519ad31c2fcee76/sklearn/experimental/enable_halving_search_cv.py>`__,
```

Comment:
Note that it's not a link to the main branch but for an old state of main where it was still in the code base. I'm fine to change but I don't think there's a real need to update these links regularly

Reply:
I know, and agree. I was just looking for `enable_hist_gradient_boosting` to remove the ones we can. Although it doesn't hurt if the link is to a file which currently exists, so that the devs can open the file in their own local env and have it the right version. Right now the file exists, and one might open it and be confused as the content is not what one expects.

Reply:
> Right now the file exists, and one might open it and be confused as the content is not what one expects.

good point